src/views/Home/Home.test.jsx

```diff
@@ -11,5 +14,47 @@ const user = {
 }
 
 test('Should render the user profile', () => {
+  render(
+    <MemoryRouter>
+      <Home user={user} />
+    </MemoryRouter>
+  );
 
+  // NAME
+  screen.getByText(/vonta/i);
+  // Motto
+  screen.getByText(/res non verba/i);
+  // Interests
+  screen.getByText(/interests/i);
+  // Avatar
+  screen.getByAltText(/avatar/i);
+  // Header
+  screen.getByAltText(/header/i);
+  // List of interests
+  const likesArr = [];
```

Comment:
This is a clever workaround!  Remember to read through the react docs and look at all the different roles you can find by like `list`.  You can grab the list directly and test it's length and contents like any array.